src/schema-templates/recipe-description.block.php

```diff
@@ -0,0 +1,15 @@
+<?php
+/**
+ * Recipe description block schema template.
+ *
+ * @package Yoast\WP\SEO\Schema_Templates
+ */
+
+use Yoast\WP\SEO\Schema_Templates\Assets\Icons;
+
+// phpcs:disable WordPress.Security.EscapeOutput -- Reason: The Icons contains safe svg.
+?>
+{{block name="yoast/recipe-description" title="<?php esc_attr_e( 'Recipe description', 'wordpress-seo-premium' ); ?>" category="yoast-required-recipe-blocks" description="<?php esc_attr_e( 'The description of the recipe.', 'wordpress-seo-premium' ); ?>" icon="<?php echo Icons::heroicons_identification(); ?>" supports={"multiple": false} }}
```

Comment:
The description and icon don't match the specs in https://docs.google.com/document/d/1YI3C30U58iGsSa3JDkG1eztS3ay-uMGqfW7HKClhRA0/edit#heading=h.cgt729sgmomg.

Reply:
The text domain shouldn't be Premium

Reply:
The category should be recommended (although if I remember correctly from the Jobs demo, in the end there will just be 1 category for all the Jobs / Recipe blocks). But for now let's make it recommended.